AstroObject.hpp

```diff
@@ -25,20 +24,21 @@ class AstroObject : public Spatial
 	//AstroObject(Rnfpp position, Rnfpp orientation, int rotationSpeed, int mass);
 	AstroObject();
 	void _init();
-	void applyForce(Rnfpp force);
+	void applyForce(Vector3 force);
```

Comment:
A general question:  why do you need all data in a variable and not a constant state?